src/ECharts/Utils.purs

```diff
@@ -0,0 +1,25 @@
+module ECharts.Utils where
+
+import Data.Argonaut.Core
+
+foreign import unnull """
+function unnull(obj) {
```

Comment:
Added comment. This function is used to copy object without null and undefined fields. 
echarts tries to get fields even if there are null or undefined.